inc/wc-calypso-bridge-mailchimp-deactivate-hook.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Clear out MailChimp tables on deactivation
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+register_deactivation_hook( 'mailchimp-for-woocommerce/mailchimp-woocommerce.php', 'wc_calypso_bridge_clear_mailchimp_tables' );
+
+function wc_calypso_bridge_clear_mailchimp_tables() {
+	global $wpdb;
+	$log = new WC_Logger();
```

Comment:
Please test for WC_Logger before using it. It is possible WooCommerce is not active when the mailchimp plugin is deactivated

Reply:
I think nothing in the plugin here is ever loaded if WC is not active [#](https://github.com/Automattic/wc-calypso-bridge/blob/master/wc-calypso-bridge-class.php#L40) - but will add a check just to be extra safe.